src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public interface Figure {
```

Comment:
You can create abstract class Figure and interfaces AreaCalculator, Drawable for methods. 
Figure can contain String color variable